vocode/streaming/action/transfer_call.py

```diff
@@ -1,10 +1,10 @@
-import os
 import aiohttp
 
 from aiohttp import BasicAuth
 from typing import Type
 from pydantic import BaseModel, Field
 
+from vocode import getenv
```

Comment:
Hi @rjheeta,

Thank you for your contribution. For consistency and best practices, please use environment variables directly instead of from vocode import getenv.

Thanks for your understanding!